packages/vm/Cargo.toml

```diff
@@ -19,6 +19,9 @@ staking = ["cosmwasm-std/staking"]
 # this enables all stargate-related functionality, including the ibc entry points
 stargate = ["cosmwasm-std/stargate"]
 # Use cranelift backend instead of singlepass. This is required for development on Windows.
+#
+# Note: This feature is automatically enabled on Windows, you don't have to toggle it manually anymore.
+# This is just to be able to debug issues with Cranelift on non-Windows platforms.
```

Comment:
I think the original comment is outdated. It is not "required for development on Windows" anymore